x-pack/plugins/alerting/server/routes/rule/apis/aggregate/transforms/transform_aggregate_body_response/v1.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
nitpick: missing `latest.ts` file

Reply:
QQ: What's the use case for using a latest fn in the http layer?

Reply:
just to follow patterns honestly :shrug: